cmd/openshift-install/create.go

```diff
@@ -137,6 +151,12 @@ var (
                           	if err != nil {
                                 	logrus.Warnf("Unable to restart ACI CNI controller")
                           	}
+
+				logrus.Infof("Expose the openshift-ingress service to Internal Loadbalancer")
```

Comment:
Updating default IngressController publish strategy to use LoadBalancerService type